src/status_im/bots/models.cljs

```diff
@@ -0,0 +1,15 @@
+(ns status-im.bots.models)
```

Comment:
This circular dependency seems like a code smell to me, is it possible to get rid of it somehow? For example, why do we have a separate `events.input` namespace? It might also be fixable with `declare` somehow. I think we should avoid having so many files for something that has one main responsibility: dealing with chat events. @flexsurfer what do you think?